packages/oui-field/src/index.spec.js

```diff
@@ -0,0 +1,355 @@
+describe("ouiField", () => {
```

Comment:
Nice tests, easy to read !

I think that testing displayed messages could be interesting :
- to validate that error message descriptions settled in provider are used and well formatted with parameters.
- to validate that only one message is displayed with correct priority

Reply:
Yes, + parameters